libqtile/core/manager.py

```diff
@@ -399,6 +399,7 @@ def update_net_desktops(self) -> None:
         # (via current_group), and when we set up the initial groups, there
         # aren't any screens yet. This can probably be changed when #475 is
         # fixed.
+        # #475 is fixed, this is still required.
```

Comment:
Is this missing a word?

Reply:
lemme rephrase

Reply:
Any chance we can just delete the bits about #475 instead? I haven't evaluated whether this TODO can be fixed, but if the part about 475 is no longer relevant, let's just remove it.